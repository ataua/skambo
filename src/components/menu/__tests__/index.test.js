import React from "react";
import renderer from "react-test-renderer";
import TopBar from "../index";
import "jest-styled-components";

import "mutationobserver-shim";

global.MutationObserver = window.MutationObserver;

jest.mock("react-redux", () => ({
  useSelector: () => jest.fn()
}))


describe("TopBar", () => {
  it("Should render menu", () => {
    const tree = renderer.create(<TopBar />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});