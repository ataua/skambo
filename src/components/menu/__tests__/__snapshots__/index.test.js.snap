// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TopBar Should render menu 1`] = `
.c0 {
  width: 100%;
  max-width: 1200px;
  height: 60px;
  position: fixed;
  z-index: 10;
  top: 0;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
}

.c1 {
  width: 15%;
  min-width: 123px;
}

.c3 {
  width: 55%;
}

.c5 {
  width: 30%;
  font-size: 32px;
  color: var(--primary);
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
}

.c2 {
  height: 50px;
  margin-top: 5px;
}

.c4 {
  width: 80%;
  top: 10px;
  left: 5px;
  cursor: pointer;
}

.c4 input {
  background: #f4f4f6 !important;
  color: var(--primary-dark) !important;
  border-radius: 30px !important;
  border: 1px solid darkgrey !important;
}

.c6 {
  width: 140px;
  height: 40px;
  margin-top: 10px;
  margin-left: 10px;
  background: var(--primary);
  color: var(--primary-light);
  font-weight: bold;
  font-size: 14px;
  border: none;
  border-radius: 5px;
  cursor: pointer;
}

.c7 {
  margin-left: 5px;
  color: var(--primary);
  background: #ffffff;
  cursor: pointer;
}

@media (max-width:500px) {
  .c1 {
    width: 15%;
    display: -webkit-box;
    display: -webkit-flex;
    display: -ms-flexbox;
    display: flex;
    -webkit-align-items: center;
    -webkit-box-align: center;
    -ms-flex-align: center;
    align-items: center;
    font-size: 30px;
  }
}

@media (max-width:500px) {
  .c3 {
    width: 65%;
    display: -webkit-box;
    display: -webkit-flex;
    display: -ms-flexbox;
    display: flex;
    -webkit-box-pack: center;
    -webkit-justify-content: center;
    -ms-flex-pack: center;
    justify-content: center;
    -webkit-align-items: center;
    -webkit-box-align: center;
    -ms-flex-align: center;
    align-items: center;
  }
}

@media (max-width:500px) {
  .c5 {
    width: 20%;
    display: -webkit-box;
    display: -webkit-flex;
    display: -ms-flexbox;
    display: flex;
    -webkit-box-pack: center;
    -webkit-justify-content: center;
    -ms-flex-pack: center;
    justify-content: center;
    -webkit-align-items: center;
    -webkit-box-align: center;
    -ms-flex-align: center;
    align-items: center;
  }
}

@media (max-width:1000px) {
  .c2 {
    height: 40px;
    margin-top: 10px;
  }
}

@media (max-width:500px) {
  .c2 {
    margin: 0;
    height: 40px;
  }
}

@media (max-width:700px) {
  .c4 {
    width: 95%;
  }
}

@media (max-width:1000px) {
  .c6 {
    margin-left: 20px;
  }
}

@media (max-width:500px) {
  .c6 {
    width: 50px;
    height: 30px;
    margin: 0;
    font-size: 10px;
    font-weight: normal;
  }
}

<div
  className="c0"
>
  <div
    className="c1"
  >
    <img
      alt="logo"
      className="c2"
      onClick={[Function]}
      src="logotipo.png"
    />
  </div>
  <div
    className="c3"
  >
    <form
      className="ui form"
      onSubmit={[Function]}
    >
      <div
        className="ui left icon input c4"
      >
        <input
          onChange={[Function]}
          placeholder="Buscar produtos para troca"
          type="text"
          value=""
        />
        <i
          aria-hidden="true"
          className="search icon"
          onClick={[Function]}
        />
      </div>
    </form>
  </div>
  <div
    className="c5"
  >
    <button
      className="c6"
      onClick={[Function]}
    >
      Entrar
    </button>
    <button
      className="c6 c7"
      onClick={[Function]}
    >
      Registrar-se
    </button>
  </div>
</div>
`;
